resources/mongo.sh

```diff
@@ -0,0 +1,28 @@
+#! /bin/sh
+
+start() {
+	LC_ALL=C mongod --bind_ip 127.0.0.1 --port 27017 \
+		--pidfilepath $SNAP_COMMON/mongod.pid \
+		--logpath=$SNAP_COMMON/mongod.log \
```

Comment:
one thing to think of is if doing this might want to make sure log rotation is on

Reply:
Ah I understood you wrong initially and turned off log rotation. 

`--logpath` automatically turns on log rotation, renames the previously opened log by appending a utc timestamp to the filename.

Reply:
good deal some how I didn't see that.  Yeah its good to keep log rotation for sure